foxai/visualizer.py

```diff
@@ -207,10 +207,17 @@ def visualize_metric(
     """
```

Comment:
To maintain coherence, could you rename the `importance_lst` parameter to `importance_list`. Maybe rename the function to something more specific - this is the function visualizing insertion/deletion metric and won't be used to display, let's say focus metric.

Reply:
Fixed.